charts/apisix-ingress-controller/crds/customresourcedefinitions.yaml

```diff
@@ -1782,6 +1782,8 @@ spec:
                     match:
                       type: object
                       properties:
+                        host:
```

Comment:
Also update the definition in apisix-ingress-controller. Cc @tao12345666333

Reply:
It's OK.